engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/DefaultOAuth2ServiceFactory.scala

```diff
@@ -33,19 +33,22 @@ class DefaultOAuth2Service[ProfileResponse: Decoder](clientApi: OAuth2ClientApi[
 
     configuration.jwt match {
       case None => profileRequestFuture
-      case Some(jwtConfiguration) => {
-        val profileFromJwtResult = new JwtValidator[ProfileResponse](jwtConfiguration).getProfileFromJwt(token)
+      case Some(jwtConfiguration) =>
```

Comment:
I thinks we can use 
case Some(jwtConfiguration) if jwtConfiguration.enabled => ...
case _ => profileRequestFuture